sparql/sparql12/expression/duplicated-values-variable.rq

```diff
@@ -0,0 +1 @@
+SELECT * WHERE { VALUES (?a ?a) { (1 1) } }
```

Comment:
I don't think this is the right place.

1. It isn't an expression.
1. All syntax tests go in a directory name of the form `syntax-*`, separate from eval tests.

Suggestion: start a new directory: `sparql/sparql12/syntax` for both positive `syntax-*.rq` tests and `syntax-bad-*.rq` for syntax tests that do not warrant their own directory.

Discussion: should this be in `sparql11`?

Reply:
Indeed, a syntax dataset would be great

> Discussion: should this be in sparql11?

imho it's undefined behavior in SPARQL 1.1 and is only defined in SPARQL 1.2 so an implementation doing something else would be still compliant with SPARQL 1.1. This is why I put this test in SPARQL 1.2.

Reply:
> This is why I put this test in SPARQL 1.2.

OK

Reply:
Moved into a new syntax dir